<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showDiagram</class>
 <widget class="QWidget" name="showDiagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#horizontalLayout
{
 background-color: rgb(23, 56, 101);

background-color: rgb(23, 56, 101);
border-radius: 7px; 
}
QWidget{
	background-color: rgb(1, 32, 79);
font: 10pt &quot;Microsoft YaHei UI&quot;;
    border-radius: 7px; 
	color: rgb(255, 255, 255);
}


QListWidget
{
background-color: rgb(23, 56, 101);
font: 10pt &quot;Microsoft YaHei UI&quot;;
border-radius: 7px; 
}

QTableWidget
{
background-color: rgb(23, 56, 101);;
	selection-background-color: rgb(85, 85, 255);
	selection-color: rgb(255, 255, 255);
font: 10pt &quot;Microsoft YaHei UI&quot;;
border-radius: 7px; 
color:rgb(255,255,255);
}
QHeaderView::section {
    color: rgb(255, 255, 255);
	background-color: rgb(36, 89, 158);
 }
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>348</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>348</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>165</width>
       <height>500</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="Start_AMU_Display" native="true">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
background-color: rgb(23, 56, 101);
font: 15pt &quot;Microsoft YaHei UI&quot;;
}

#Start_AMU_Display{border:2px solid rgb(255, 255, 255);font: 700 16pt &quot;Microsoft YaHei UI&quot;;}
#title_4{
	color: rgb(255, 255, 255);
}
#line_5
{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="3,0,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="title_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>92</width>
              <height>23</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>起始质谱</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#line_5
{
background-color: rgb(255, 255, 255);
}

#line_5::disabled
{
background-color: rgb(158, 158, 158);
}</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Start_AMU_Value">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);


</string>
            </property>
            <property name="text">
             <string>0AMU</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="Stop_AMU_Display" native="true">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
background-color: rgb(23, 56, 101);
font: 15pt &quot;Microsoft YaHei UI&quot;;
}

#Stop_AMU_Display{border:2px solid rgb(255, 255, 255);font: 700 16pt &quot;Microsoft YaHei UI&quot;;}
#title_5{
	color: rgb(255, 255, 255);
}
#line_6
{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10" stretch="3,0,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="title_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>92</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>停止质谱</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_6">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#line_5
{
background-color: rgb(255, 255, 255);
}

#line_5::disabled
{
background-color: rgb(158, 158, 158);
}</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Stop_AMU_Value">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);


</string>
            </property>
            <property name="text">
             <string>0AMU</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="Ppamu_Display" native="true">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
background-color: rgb(23, 56, 101);
font: 15pt &quot;Microsoft YaHei UI&quot;;
}

#Ppamu_Display{border:2px solid rgb(255, 255, 255);font: 700 16pt &quot;Microsoft YaHei UI&quot;;}
#title_6{
	color: rgb(255, 255, 255);
}
#line_7
{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11" stretch="3,0,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="title_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>92</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>分辨率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#line_5
{
background-color: rgb(255, 255, 255);
}

#line_5::disabled
{
background-color: rgb(158, 158, 158);
}</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Ppamu_Value">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);


</string>
            </property>
            <property name="text">
             <string>0PPAMU</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>510</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>610</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
